Bottom: 6770f575ea01aea475f29c0c854ff94913156e7d
Top:    ea8d242863d25ef4054585452aec120d3bce1486
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-05-21 13:47:55 -0400

Add a reset function


---

diff --git a/arch/arm/mach-lpc31xx/board-ea313x.c b/arch/arm/mach-lpc31xx/board-ea313x.c
index b7864a4..0e537ec 100644
--- a/arch/arm/mach-lpc31xx/board-ea313x.c
+++ b/arch/arm/mach-lpc31xx/board-ea313x.c
@@ -47,6 +47,7 @@
 #include <mach/gpio.h>
 #include <mach/i2c.h>
 #include <mach/board.h>
+#include <mach/system.h>
 
 static struct lpc313x_mci_irq_data irq_data = {
 	.irq = IRQ_SDMMC_CD,
@@ -735,6 +736,7 @@ MACHINE_START(EA313X, "NXP EA313X")
 	.init_irq	= lpc313x_init_irq,
 	.timer		= &lpc313x_timer,
 	.init_machine	= ea313x_init,
+	.restart	= arch_reset,
 MACHINE_END
 #endif
